docs/databases.rst

```diff
@@ -0,0 +1,149 @@
+Configuring SQL Databases in CommCare
+=====================================
+
+CommCare makes use of a number of logically different SQL databases. These databases
+can be all be a single physical database or configured as individual databases.
+
+By default CommCare will use the `default` Django database for all SQL data.
+
+.. image:: images/django_db_monolith.png
+
+Synlog Data
+~~~~~~~~~~~
+Synclog data may be stored in a separate database specified by the
+`SYNCLOGS_SQL_DB_ALIAS` setting. The value of this setting must be a DB
+alias in the Django `DATABASES` setting.
+
+UCR Data
+~~~~~~~~
+Data created by the UCR framework can be stored in multiple separate databases.
```

Comment:
nit: might be nice to explicitly mention `corehq.apps.userreports` or link to the readme or something in case someone is not familiar with UCR

Reply:
```suggestion
Data created by the `UCR <ucr.html>`_ framework can be stored in multiple separate databases.
```